modules/oadp-using-data-mover-for-csi-snapshots.adoc

```diff
@@ -84,18 +116,47 @@ spec:
         default: true
         objectStorage:
           bucket: <bucket_name>
-          prefix: <bucket_prefix>
+          prefix: <bucket-prefix>
         provider: aws
   configuration:
     restic:
       enable: <true_or_false>
     velero:
+       itemOperationSyncFrequency: "10s"
```

Comment:
@CarmiWisemon can you please fix the Indentation ? I'm seeing defaultPluginList field highlighted in red color in doc previous. Thanks in advance

Reply:
Done. Thanks for noticing.